src/components/ProgressBar/ProgressBar.scss

```diff
@@ -20,7 +20,7 @@
 .ProgressBar {
   overflow: hidden;
   width: 100%;
-  background-color: color('sky', 'base');
+  background-color: var(--p-neutral-action, color('sky'));
```

Comment:
```suggestion
  background-color: var(--p-surface-background, color('sky'));
```

Since this element is not interactive, let's use the surface background value for it.

Reply:
I was originally going to use `surface-background` however when `ProgressBar` is used on a page, without a card/modal/etc it blends in. 
![Screen Recording 2019-12-09 at 01 24 PM](https://user-images.githubusercontent.com/24610840/70462017-02055600-1a88-11ea-8f22-7a9b30def3df.gif)

Is this intended?

Reply:
Good point.

Using the disabled neutral action for the background ought to do the trick. We've used this in `badge` as well for a similar application.